book/website/reproducible-research/open/open-access.md

```diff
@@ -44,10 +44,12 @@ The number of these journals is still increasing rapidly, with 17,000 listed in
 
 Watch this talk by Vicky Hellon on "Brief History of scientific publishing, and where do we go from here?", where she provides an overview of how different models of scientific publication look like, and how we can take an informed approach to Open Access.
 
-```{iframe} https://www.youtube.com/embed/Mycp7SzBWB0
+::: {iframe} https://www.youtube.com/embed/Mycp7SzBWB0
 :align: center
 :width: 100%
-History of  Scientific Publishing - Where do we go from here?? from [ORCID](https://vimeo.com/orcidvideos) on [Vimeo](https://vimeo.com).
+
+History of Scientific Publishing - Where do we go from here?? from [ORCID](https://vimeo.com/orcidvideos) on [Vimeo](https://vimeo.com).
```

Comment:
Looking through this in review, I think this caption was actually wrong - let's fix it.
```suggestion
History of Scientific Publishing - Where do we go from here??.
```